src/triggers.c

```diff
@@ -932,12 +932,24 @@ void hurt_on()
 	self->s.v.solid = SOLID_TRIGGER;
 	self->s.v.nextthink = -1;
 	setorigin(self, PASSVEC3(self->s.v.origin)); // it matter
+	g_globalvars.force_retouch = 2; // make sure even still objects get hit
 }
 
 void hurt_touch()
 {
 	if (!other->s.v.takedamage)
 	{
+		if (cvar("k_ctf_hurt_items"))
```

Comment:
I don't really like this, nano...

There is this function, hurt_tocuh(), which has a purpose. And now, we inject into it another functionality...
On top of this, this new functionality (about the runes) is only triggered if we are actually trying to get out from the function without it doing anything (when other->s.v.takedamage is zero).

Could you come up with a more elegant solution?
At least move this ctf-rune related code (the stuff you've just added) to a function, and call the function here.

Reply:
I'll move it.

Another approach would be to implement actual health and damage to dropped runes and flags but that felt like a too big scope to start with as it would have greater impact on gameplay mechanics - in a possibly interesting way.

Reply:
But even if it different behavior from damage, I think the most common goal of trigger_hurt in multiplayer maps is to delete whatever touches it. Btw, wrt move, function in same file? Otherwise input on if it should be both in runes.c and ctf.c, and if config var in that case should be split.

Reply:
> But even if it different behavior from damage, I think the most common goal of trigger_hurt in multiplayer maps is to delete whatever touches it. 
You are right, it is a trigger and it is a touch function of it, could be any logic, definitely better than messing with health and T_Damage().